packages/core/src/App/Containers/CitizenshipModal/same-citizenship-modal.jsx

```diff
@@ -9,10 +9,11 @@ const SameCitizenshipModal = ({
     setIsCitizenshipModal,
     setIsPasswordModal,
     setIsSameCitizenshipModal,
+    selectedCountry,
 }) => (
     <div>
         <Text as='p' className='account-signup__text'>
-            {localize('Are you a citizen of the same country where you live?')}
+            {localize('Are you a citizen of ')} {selectedCountry}?
```

Comment:
1. Can you use `residence` value?
2. You need to make sure selected country is being translated using `localize`

Reply:
Will do it 🙏